### PR TITLE
Solve issue #1318 adding Append method to kratos_parameters

### PR DESCRIPTION
**Pull request by josep-m-carbonell**
Append method for Array is added  to json parameters

### CODE REVIEW

File: kratos/python/add_kratos_parameters_to_python.cpp [hunk 1]
```diff
@@ -108,6 +108,7 @@ void  AddKratosParametersToPython()
     .def("__getitem__", &Parameters::GetValue)
     .def("__setitem__", &Parameters::SetArrayItem)
     .def("__getitem__", &Parameters::GetArrayItem)
+    .def("PushBack", &Parameters::PushBack)
```
Comment (RiccardoRossi) [inline kratos/python/add_kratos_parameters_to_python.cpp]: in python, the "append" command is used to append to a list. I wonder if we should not follow the python style on this. 

To make it clear i am posting this as a question, not as a suggestion

File: kratos/includes/kratos_parameters.h [hunk 2]
```diff
@@ -383,8 +383,8 @@ class Parameters
     {
         rapidjson::Value tmp(value.c_str(), mpdoc->GetAllocator());
         *mpvalue = tmp;
-//         mpvalue->SetString(rapidjson::StringRef(value.c_str()));
-//        mpvalue->SetString(value.c_str(), value.length());
+	//mpvalue->SetString(rapidjson::StringRef(value.c_str()));
```
Comment (RiccardoRossi) [inline kratos/includes/kratos_parameters.h]: minor: we could directly take out these commented lines

File: kratos/python/add_kratos_parameters_to_python.cpp [hunk 3]
```diff
@@ -78,7 +95,7 @@ void  AddKratosParametersToPython()
     .def("ValidateAndAssignDefaults",&Parameters::ValidateAndAssignDefaults)
```
Comment (RiccardoRossi) [inline kratos/python/add_kratos_parameters_to_python.cpp]: minor: just before merging, you could pass the clang-format (or astyle) on these files.

**Review by RiccardoRossi — DISMISSED** [review]

File: kratos/includes/kratos_parameters.h (hunk 2 above)
Comment (josep-m-carbonell) [reply kratos/includes/kratos_parameters.h]: ok

File: kratos/python/add_kratos_parameters_to_python.cpp (hunk 3 above)
Comment (josep-m-carbonell) [reply kratos/python/add_kratos_parameters_to_python.cpp]: If I merge the new format files, the pull-request needs to be approved again. Ok

**Review by RiccardoRossi — APPROVED** [review]

**Pull request merged**